packages/ember-core/src/components/tooltip.gts

```diff
@@ -80,6 +81,7 @@ const Tooltip: TOC<TooltipSignature> = <template>
   <Popover
     class="tooltip"
     @alignment={{@alignment}}
+    @delay={{@delay}}
```

Comment:
Should a tooltip have a default delay? Based on those articles you sent me before, a default delay of 300ms seems most appropriate.